apps/api/src/app/workflows-v2/usecases/get-step-schema/get-step-metadata-use-case.service.ts

```diff
@@ -0,0 +1,150 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
```

Comment:
Let's rename this to just get-step-schema use case to align with the controller. Metadata is a very broad term and creates confusion. In fact, the returned data are the step definition, not metadata about a step.

Reply:
changing to get step data ok? it's not only schmas, also the controler moved to the workflow controler